packages/design-system-react/jest.setup.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-unassigned-import
+import '@testing-library/jest-dom';
```

Comment:
Disables this linting rule `error  Imported module should be assigned`

<img width="405" alt="Screenshot 2024-10-31 at 3 04 49 PM" src="https://github.com/user-attachments/assets/1de8cce2-fd25-40c2-bef2-9904d7437e88">